angular2-webpack-starter/src/app/pages/login/login.component.html

```diff
@@ -0,0 +1,15 @@
+<!--  LOGIN  -->
+<div id="login" class="login" data-src="assets/img/big-images/img1.jpg" background>
+    <div [ngClass]="errorLogin" class="error">Error: Username or password is incorrect</div>
+    <!--  CIRCLE  -->
+    <form class="circle" name="form" (ngSubmit)="f.form.valid && login()" #f="ngForm">
```

Comment:
you can use `authenticationService.login()` all logic should be storage in services